src/types/profile/settings.rs

```diff
@@ -26,7 +26,12 @@ pub struct Settings {
     pub subtitles_text_color: String,
     pub subtitles_background_color: String,
     pub subtitles_outline_color: String,
+    /// Whether or not the Escape key should exists from the app when in Full screen.
+    pub esc_exists_fullscreen: bool,
+    /// The Seek time duration (in milliseconds) is when using the Arrow keys
     pub seek_time_duration: u32,
+    /// The Seek shift time duration (in milliseconds) is when using the Arrow keys + Shift
+    pub seek_shift_time_duration: u32,
```

Comment:
Instead of changing logic for the whole variable and migrating it, wouldn't it be better for the new variable to be different ie. `seek_long_time_duration`. As it will create an easy error, where the old variable name stays the same, but it's value is changed. Since `seek_time_duration` was meant to be used as the default seek time with the default method of seeking. 
Now it's just hardcoding logic in settings which is meant for web only. (there's no shift on mobile or tablet or tvs). And then IMO it should be a completely new setting for web only, which doesn't impact any other services.

Reply:
The old variable is the long one, using the Shift key is the finer seeking.
I'm open for a new name.

Reply:
This means that we still need to migrate the value as that's the default seeking that we want to change in a sense.

Reply:
as discussed on slack, I've update the PR and initial comment to reflect the discussion.
I'm resolving this conversation.